README.Wii

```diff
@@ -0,0 +1,47 @@
+# SDL port to Nintendo GameCube/Wii
+
+This port uses the libogc library and is meant to be built with the devkitPPC
+toolchain, available from http://www.devkitpro.org
+
+
+## Installation instructions
+
+It is strongly recommended to only use the pre-built packages available from
```

Comment:
The language here seems unnecessary. Replace with 
`We highly recommend using pre-built packages provided via [pacman](https://devkitpro.org/wiki/devkitPro_pacman) unless intending to contribute changes to this port.`